spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/config/parameterstore/ParameterStoreConfigDataLoaderIntegrationTests.java

```diff
@@ -116,8 +117,10 @@ void whenKeysAreNotSpecifiedFailsWithHumanReadableFailureMessage(CapturedOutput
 			assertThat(e).isInstanceOf(ParameterStoreKeysMissingException.class);
 			// ensure that failure analyzer catches the exception and provides meaningful
 			// error message
-			assertThat(output.getOut())
-					.contains("Description:\n" + "\n" + "Could not import properties from AWS Parameter Store");
+			// Ensure that new line character should be platform independent
```

Comment:
This may or may not be relevant as project build is running on linux system so test will never fail in the build. But as my dev machine is windows I faced so think of fixing it.